packages/census/scripts/modify-snarkjs-package.js

```diff
@@ -0,0 +1,19 @@
+const fs = require('fs');
+const path = require('path');
+
+let packagePath = path.dirname(require.resolve("snarkjs"));
+
+while(!fs.existsSync(packagePath + '/package.json')) {
```

Comment:
Using these explicit slashes will work in Unix/Linux, but not Windows. Most likely not an effective issue, but I would use `path.sep` instead.